packages/orchestration/src/ethereum-api.ts

```diff
@@ -1,7 +1,10 @@
+import type { BaseChainInfo } from './orchestration-api.ts';
+
 /**
  * Info for an Ethereum-based chain.
  */
-export type EthChainInfo = Readonly<{
-  chainId: string;
-  allegedName: string;
-}>;
+export interface EIP155ChainInfo extends BaseChainInfo {
+  namespace: 'eip155';
+  /** namespace + reference required for non-cosmos chains. eip155 chain references are always natural numbers */
+  chainId: `eip155:${number}`;
```

Comment:
shouldn't it then be … ? 
```suggestion
  /** eip155 chain references are always natural numbers */
  reference: number;
```

That that change was missed supports the argument against including a redundant computed field.